src/shadow-embed.js

```diff
@@ -61,7 +62,10 @@ export function createShadowRoot(hostElement) {
   // Native support.
   const shadowDomSupported = getShadowDomSupportedVersion();
   if (shadowDomSupported == ShadowDomVersion.V1) {
-    return hostElement.attachShadow({mode: 'open'});
+    // NOTE(torch2424) need to have an id on the shadow root no matter what
+    const shadowRoot = hostElement.attachShadow({mode: 'open'});
```

Comment:
I tried moving this into `installStylesForShadowRoot()` however, I realized that it is called multiple times to generate our css encapsulation, and cannot have the random id generated within it. Therefore to solve that, we would have to pass the id into the call, which would make things awkward in my opinion. I cleaned up the implementation of adding the id, but if you have a better suggestion, please let me know